var/spack/repos/builtin/packages/py-hatchling-vcs/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyHatchlingVcs(PythonPackage):
```

Comment:
```suggestion
class PyHatchVcs(PythonPackage):
```
Same with directory name